src/utils/corps.ts

```diff
@@ -51,7 +52,7 @@ export const displayNumber = (
     ? `#${corps.number}`
     : corps.bNumber
     ? `b${corps.bNumber}`
-    : 'p.e.';
```

Comment:
`displayNumber` is used for other things than this and should not be changed, as it will break other functions.